packages/suite/src/components/wallet/TransactionItem/TransactionTarget/TransactionTarget.tsx

```diff
@@ -37,6 +41,17 @@ export const TokenTransfer = ({
     isPhishingTransaction,
     ...baseLayoutProps
 }: TokenTransferProps) => {
+    const fiatCurrencyCode = useSelector(selectLocalCurrency);
+    const fiatRateKey = getFiatRateKey(
+        transaction.symbol,
+        fiatCurrencyCode,
+        transaction.tokens[0]?.contract as TokenAddress,
```

Comment:
This component is for specific token transfer, you have to use `transfer.contract`, not first token from transaction, there can be more different tokens in one tx

```suggestion
        transfer.contract as TokenAddress,
```